src/attendee/api/views.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
+from rest_framework import views, status
+from rest_framework.response import Response
+
+from registry.helper import reg
+from ext2020.models import Attendee
```

Comment:
sort the imports by:

1. native packages
2. 3rd party libraries
3. customized packages

and use an empty line to separate them.

```suggestion
from registry.helper import reg

from ext2020.models import Attendee
```

Reply:
`ext2020` is going to be deprecated. Please remove the dependencies on this app.